specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-08-01-preview/workspaceRP.json

```diff
@@ -2797,8 +2858,8 @@
       }
     },
     "PrivateLinkServiceConnectionState": {
```

Comment:
>PrivateLinkServiceConnectionState [](http://example.com/codeflow?start=5&length=33)

Does this conform to the common-types private link service connection state definition? Can it be changed to a $ref? 

[https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L65](https://github.com/Azure/azure-rest-api-specs/blob/5dd1107d5f2be8d600325d795450e1d854fbe7e8/specification/common-types/resource-management/v5/privatelinks.json#L65)

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2688199